docs/source/install.rst

```diff
@@ -54,6 +54,10 @@ Cap will search for the following libraries at configuration-time:
 
     * Boost.MPI
 
+    * Boost.Python (if you enabled Cap with Python)
```

Comment:
Should add iostream, system, and thread. Otherwise, deal.II could use a different version of boost